src/Workspaces/CSharp/Portable/Extensions/ContextQuery/SyntaxTreeExtensions.cs

```diff
@@ -1039,6 +1040,24 @@ public static bool IsDefaultExpressionContext(this SyntaxTree syntaxTree, int po
                 return true;
             }
 
+            if (allowedModifiersBefore != null &&
```

Comment:
Why? the first case is for when the token before is an open paren. No modifiers can exist then?

---
In reply to: [120768444](https://github.com/dotnet/roslyn/pull/19703#discussion_r120768444) [](ancestors = 120768444)